src/lib.rs

```diff
@@ -1,11 +1,22 @@
 #![deny(missing_docs)]
 #![deny(missing_debug_implementations)]
+#![deny(clippy::self_named_module_files)]
 #![cfg_attr(test, deny(rust_2018_idioms))]
 #![cfg_attr(all(test, feature = "full"), deny(unreachable_pub))]
 #![cfg_attr(all(test, feature = "full"), deny(warnings))]
 #![cfg_attr(all(test, feature = "nightly"), feature(test))]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
+#![allow(
+    clippy::needless_borrow,
+    clippy::module_inception,
+    clippy::useless_format,
+    clippy::redundant_clone,
+    clippy::manual_non_exhaustive,
+    dead_code,
+    unused_imports
```

Comment:
```
warning: unused import: `self::body::Sender`
  --> src/body/mod.rs:24:16
   |
24 | pub(crate) use self::body::Sender;
   |                ^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

```

Reply:
Note that `unused_imports` lint is from `rustc` itself, and not clippy. Also, I think it's best to address both `unused_imports` and `dead_code` lint.